### ISSUE TITLE
remove imports from surfaces app

### ISSUE THREAD

**Issue by ltalirz** [aiidalab/aiidalab-aiida#4]
The workflow manager currently imports workflows from the `mc-surfaces` app.

https://github.com/materialscloud-org/mc-aiida/blame/master/browse_workflows.ipynb#L35-L37

We should definitely try to find a way around this - people can uninstall the surfaces app, and it is anyhow not scalable if we need to modify the workflows app for any new workflow that becomes available.

I'm not a workflow's expert, so perhaps someone can tell me why these imports are needed?
The `NanoribbonWorkChain` and `SlabGeoOptWorkChain` are not used anywhere in the code...

**Comment by oschuett** [comment]
Yes, it's workaround for this [issue](https://github.com/aiidateam/aiida_core/issues/975). It's needed for the resubmit feature.
